include/picongpu/fields/incidentField/profiles/GaussianBeam.hpp

```diff
@@ -251,7 +253,9 @@ namespace picongpu
                             // inverse radius of curvature of the beam's  wavefronts
                             auto const R_inv = -focusPos / (Unitless::R * Unitless::R + focusPos * focusPos);
                             // the Gouy phase shift
-                            auto const xi = math::atan(-focusPos / Unitless::R);
+                            auto xi = math::atan(-focusPos / Unitless::R);
+                            if(simDim == DIM2)
+                                xi *= 0.5_X;

```

Comment:
This looks like a bug fix for the already existing profiles.
We should separate these changes to be able to write a clean changelog.

Reply:
It seems to be a fix of #4249 . Indeed, should be in a separate PR.

Reply:
See #4301